pages/AlltimeFavorites/index.js

```diff
@@ -0,0 +1,126 @@
+import PreviewPicture from "../../components/PreviewPicture";
+import { Container } from "../../components/styledComponents/Container";
+import { StyledButton } from "@/components/styledComponents/StyledButton";
+import React from "react";
+import useSWR from "swr";
+import Loading from "../../components/Loading";
+import { useAtom } from "jotai";
+import { isFavorit } from "../../store/isFavorit";
```

Comment:
```suggestion
import { isFavorite } from "../../store/isFavorite";
```

Reply:
changed to isFavoriteState to avoid confusion with isFavorite boolean in DB